### PR TITLE
Ts and js

### PR DESCRIPTION
**Pull request by Skarpej**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,26 +4,4 @@ Linting tools are used for identifying and reporting on _code quality_ and _styl
...
-
-## Prettier is a code formatter, not a linter
-
-[Prettier](https://www.npmjs.com/package/prettier) is a code formatter that takes care of formatting your code according to a certain style. However, it doesn't handle _code quality rules_ (which linters do). Linters handle both _code style_ and _code quality_ means that if you're using both a linter and Prettier, there's a risk of Prettier and the linter colliding. There are ways to combine Prettier and linters in efficiently and safely, but since we want you to be fully confident with linters we suggest that you refrain from using Prettier during the salt bootcamp.
-
-## Conclusion
-
-Linting is important, especially in a professional environment. We expect you to be comfortable with the linting setup described in this repository, so go ahead and start implementing it in all your projects!
-    
+Before you start, you need to know if you are working with TypeScript or JavaScript, and work from that folder. So chose the correct and folder we will see you there!
```
Comment (AhsanAyaz) [inline README.md]: Can we add the links to the readmes in the folders here as well?

**Review by AhsanAyaz — COMMENTED** [review]
Added a minor comment. The rest of it looks good.

File: README.md (hunk 1 above)
Comment (Skarpej) [reply README.md]: Great idea! Will do

Comment (ic3Dragon) [reply README.md]: I think it should be "choose the correct folder and we ..." ?

File: README.md [hunk 2]
```diff
@@ -4,4 +4,4 @@ Linting tools are used for identifying and reporting on _code quality_ and _styl
 
 ## Linting at Salt
 
-Before you start, you need to know if you are working with TypeScript or JavaScript, and work from that folder. So chose the correct and folder we will see you there!
+Before you start, you need to know if you are working with [JavaScript](./JavaScript/README.md) or [TypeScript](./TypeScript/README.md), and work from that folder. So choose the correct folder and folder we will see you there!
```
Comment (ic3Dragon) [inline README.md:7]: There's still a small typo here. It should be : 
So choose the correct folder and we will see you there!

**Review by AhsanAyaz — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**